python/search.py

```diff
@@ -17,7 +16,7 @@
 # tab of
 #   https://cloud.google.com/console
 # Please ensure that you have enabled the YouTube Data API for your project.
-DEVELOPER_KEY = 'REPLACE_ME'
+DEVELOPER_KEY = 'AIzaSyAeoEyavpe_6Lcy3S46yeYUApRdGXgM6w4'
```

Comment:
I strongly believe that this information is very sensitive, I suggest you remove your KEY and replace it with a placeholder and/or use some library to manage secrets and environment variables